test/elixir-mode-indentation-test.el

```diff
@@ -1947,6 +1947,25 @@ defmodule Test do
 end
 ")
 
+(elixir-def-indentation-test indent-before-comma
+                             (:tags '(indentation))
+"
+defmodule TestIndentation do
+import Enum
+
```

Comment:
The problem actually occurred only when there was no newline here (after `import Enum`).